### PR TITLE
Clean/Fix Punctuation

### PR DESCRIPTION
**Pull request by robUx4**
Similar to RFC Editor reviews

### CODE REVIEW

File: codec_specs.md [hunk 1]
```diff
@@ -812,7 +812,7 @@ Codec ID: S_TEXT/SSA
 Codec Name: Subtitles Format
 
 Description: The [Script Info] and [V4 Styles] sections are stored in the codecprivate. Each event is stored in its own Block.
-For more information, see [the SSA/ASS section in Subtitles](#subtitles).
+For more information; see [the SSA/ASS section in Subtitles](#subtitles).
```
Comment (mbunkus) [inline codec_specs.md:815]: This change looks wrong. When in doubt if you should use a comma or a semicolon try using a full stop instead. If the result still makes sense, you use a semicolon. And that's not the case here: "For more information. See URL." is not OK.

I'm not going to add a comment for each of these "For more information, see…" changes, but they're all wrong IMHO.

File: index_matroska.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/)
 But since it's quite complete it is used as a reference for the development of libmatroska.
 A legacy version of the specification can be found [here](https://www.matroska.org/files/matroska_file_format_alexander_noe.pdf) (PDF doc by Alexander Noé -- outdated).
 
-For a simplified diagram of the layout of a Matroska file, see the [Diagram page](diagram.md).
+For a simplified diagram of the layout of a Matroska file; see the [Diagram page](diagram.md).
```
Comment (mbunkus) [inline index_matroska.md:81]: Same argument as above: the "replace with full stop" test fails. A comma is appropriate, a semicolon isn't.

File: index_matroska.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ Specific terms are defined below:
 
 Matroska is a Document Type of EBML (Extensible Binary Meta Language).
 This specification is dependent on the [EBML Specification](https://github.com/Matroska-Org/ebml-specification/blob/master/specification.markdown).
-For an understanding of Matroska's EBML Schema, see in particular the sections of the EBML Specification covering
+For an understanding of Matroska's EBML Schema; see in particular the sections of the EBML Specification covering
```
Comment (mbunkus) [inline index_matroska.md:131]: Same as above; please revert.

**Comment by robUx4** [comment]
Rebased and removed the bogus changes noted by @mbunkus

**Pull request merged**